content/docs/installation/helm.md

```diff
@@ -282,7 +282,100 @@ Generally we recommend:
 You may want to consider your approach along with other tools that may offer
 helm compatible installs, for a standardized approach to managing CRD
 resources. If you have an approach that cert-manager does not currently
-support, then please 
+support, then please
 [raise an issue](https://github.com/cert-manager/cert-manager/issues) to
 discuss.
 
+
+## Install the Helm chart using Flux
```

Comment:
Shouldn't we add a separate Flux section?

Reply:
When I go to cert-manager.io, I probably would look for a Flux section if that is the tool I'm using.

Reply:
What precisely do you propose?

I did consider adding a `flux.md` page, but decided that we'd end up with too many submenu items if we also add `argo.md`, `anthos.md`, `octopus.md` etc.

I also considered that Flux can install cert-manager in other ways too, so it seems appropriate to document the Flux Helm Controller here and in future document the Flex Kustomize controller in the kubectl applyt page.

I looked for examples from other projects and I found that Kyverno have a https://kyverno.io/docs/installation/platform-notes/ page where they include notes for ArgoCD users.
Couldn't find any other examples of projects that have a specific page for installing using flux or argo.

I have tried creating a "Continuous Deployment" page and move this tutorial there.
See what you think.

Reply:
What do you think about the name "GitOps"?

Reply:
I don't think it's appropriate. 

GitOps is about storing your infrastructure configuration in a Git repo and then using a continuous delivery / continuous deployment like Flux tool to apply it.
So this page is about the CD part. 

Why do you suggest it?

Reply:
Also GitOps is the brand name Red Hat uses on their [ArgoCD hybrid](https://www.redhat.com/en/technologies/cloud-computing/openshift/gitops), so I would stay FAR away from that term. 😉